src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -105,20 +105,37 @@ def default_api_version(self):
         ResourceType.MGMT_STORAGE: '2016-01-01',
         ResourceType.MGMT_NETWORK: '2015-06-15',
         ResourceType.MGMT_COMPUTE: SDKProfile('2016-03-30'),
-        ResourceType.MGMT_RESOURCE_FEATURES: '2015-12-01',
         ResourceType.MGMT_RESOURCE_LINKS: '2016-09-01',
         ResourceType.MGMT_RESOURCE_LOCKS: '2015-01-01',
         ResourceType.MGMT_RESOURCE_POLICY: '2015-10-01-preview',
         ResourceType.MGMT_RESOURCE_RESOURCES: '2016-02-01',
         ResourceType.MGMT_RESOURCE_SUBSCRIPTIONS: '2016-06-01',
         ResourceType.DATA_STORAGE: '2015-04-05',
+        ResourceType.MGMT_KEYVAULT: '2016-10-01',
```

Comment:
This KV should have already existed with Scott's PR?

Reply:
I rebased everything on latest dev branch, so not sure why it's showing up here.